### ISSUE TITLE
increasing redirects (base64 & zoneid) not being blocked

### ISSUE THREAD

**Issue by Rokeugon** [gorhill/uBlock#2135]
Read first: <https://github.com/gorhill/uBlock/blob/master/CONTRIBUTING.md>

### Describe the issue

plenteful of streaming sites im now using now-a-days are doing redirects to users that simply click on the webpage. thankfully i have javascript disabled automatically for all websites i visit, but the fact i still get a cookie while visiting the site is an annoying process

### One or more specific URLs where the issue occurs

im going to suggest a site that has known to be malware infested so please test at your own risk. if you know what youre doing then just be careful :)

123movies.gs or .is .ru etc etc they have a bunch of proxy domains.

which i dunno how but it triggers this for some unknown reason

[URL(s) for issue on a specific site are **mandatory**]

### Screenshot in which the issue can be seen

i mean sure id supply it but dunno how exactly thats be of any help really.

[Screenshot(s) for visual issues are **mandatory**]

### Steps for anyone to reproduce the issue

again please test at your own discretion!
*HAVE JAVASCRIPT DISABLED BY DEFAULT PLEASE FOR ALL WEBSITES*

1. load webpage
2. click anywhere on webpage
3. upon clicking a new tab will open meaning youve been redirected

### Your settings

http://i.imgur.com/LCttzJp.png

Chrome: Version 54.0.2840.99 m
uBlock Origin v1.9.16

##### Your filter lists

http://i.imgur.com/fEASZNa.png

http://i.imgur.com/5AXf9p0.png

Extra Thoughts:

basically i dunno how but either a base64 encoded link is being delivered and unpacked that then gives a zoneid link which you have 1 domain blocked as shown above, it will move onto the next string of domains to deliver to you. every 2-3 days i revist the site there is always a new redirect. for example: todays redirect was going to here

http://dashgreen.online/?zoneid=824944&pbk2=8be9aedeeeae924e46bf5f0b913533c76351099718386163281&r=%2Foc%2Fhan&ab=118&uuid=b98bac32-4179-429b-bc97-20a3149328d5

**Comment by gorhill** [comment]
Closing and marking as _invalid_, as per [CONTRIBUTING](https://github.com/gorhill/uBlock/blob/master/CONTRIBUTING.md).

Also, I am not interested in visiting "a site that has known to be malware infested" -- you should report that site to the malware domains lists maintainers.

**Comment by Rokeugon** [comment]
@gorhill well i mentioned that the site has been "Known" to be infected with malware. but the particiular domain is clean

https://sitecheck.sucuri.net/results/123movies.gs

i only posted this to see why we dont all ready have base64 and zoneid blocking, even tho some filters say they do they clearly dont.

might not be a severe problem for me but for others that are not as prepared might have issues, this problem could lead to malicious intent with the amount of new tabs that open and redirects that happen.

**Comment by gorhill** [comment]
It's a filter issue, not a uBO's one. Use the logger to find the chain of redirects which open the popups (select_All_ in the logger's dropdown list). Also, did you try to use the [no-popups per-site switch](https://github.com/gorhill/uBlock/wiki/Per-site-switches#no-popups)?

**Comment by Rokeugon** [comment]
yes ive tried enabling it, but unfortunately the issues still persist. as for the logger, il give it another go and sift through everything to try find whats responsible

**Comment by gorhill** [comment]
I just turned the no-popup switch for that site and the popup was automatically closed, no new filters needed. However, the site is using websockets for quite probably not friendly purpose, block them with:

    *$websocket,domain=123movies.gs|123movies.is

Filter issues must be reported on EasyList et al, or at uAssets in the event something cannot be blocked using standard ABP filter syntax.
